Objective-C/Tests/LogTest.m

```diff
@@ -121,7 +90,8 @@ - (void) writeAllLogs: (NSString*)string {
     CBLWarnError(Database, @"%@", string);
 }
 
-- (BOOL) isKeywordPresentInAnyLog: (NSString*)keyword path: (NSString*)path {
+- (BOOL) isKeywordPresentInAnyLog: (NSString*)keyword path: (nullable NSString*)path {
+    if (!path) return NO;
```

Comment:
What is the case that this function is called with path == nil? If there is a case, maybe it's more straightforward to just assert the path in that test as it doesn't seem like it's responsibility of this method to verify the path?

Reply:
I will remove this, alongside test `testFileLoggingDisabled` because it is redundant. The only check we can do is that .directory is nil when the .file is nil... which is always true. We already set it to nil in the next test `testFileLoggingReEnableLogging` so I will move the assert for .directory there.

Reply:
I will keep it for the old test because there we do set level None, but we actually create a file log